buildSrc/src/main/kotlin/nessie-conventions.gradle.kts

```diff
@@ -160,6 +160,7 @@ fun Project.configureJava() {
     if (System.getProperty("withMavenLocal").toBoolean()) {
       mavenLocal()
     }
+    maven { setUrl("https://repository.apache.org/content/repositories/orgapacheiceberg-1088/") }
```

Comment:
Move that up before `mavenCentral` and add a `content` + `include` rule to only include Iceberg.

Reply:
And give it a `name`

Reply:
this will be removed once 0.13.2 has been released